starter-code/index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = []
+    this.length = 0
```

Comment:
la longitud se esperaba que fuera la longitud del los items, es decir: this.length = this.items.length para luego en el de suma no tener que ir sumandolo